kzg_multi_proofs_test.go

```diff
@@ -12,16 +11,16 @@ func TestKateSettings_CheckProofMulti(t *testing.T) {
 	s1, s2 := generateSetup("1927409816240961209460912649124", 16+1)
 	ks := NewKateSettings(fs, s1, s2)
 	for i := 0; i < len(ks.secretG1); i++ {
-		t.Logf("secret g1 %d: %s", i, strG1(&ks.secretG1[i]))
+		// t.Logf("secret g1 %d: %s", i, strG1(&ks.secretG1[i]))
```

Comment:
The logs are here so I can compare double-check the values produced by the python implementation. See https://github.com/ethereum/research/tree/master/kzg_data_availability
Just keep them there, benchmarks are separate.